civis/io/_tables.py

```diff
@@ -33,6 +34,36 @@
 }
 
 
+SQL_PANDAS_MAP = {'smallint': np.int,
+               'int': np.int,
+               'integer': np.int,
+               'bigint': np.int,
+               'decimal': np.float,
+               'numeric': np.float,
+               'float': np.float,
+               'float4': np.float,
+               'float8': np.float,
+               'real': np.float,
+               'double precision': np.float,
+               'boolean': np.bool,
+               'bool': np.bool,
+               'int2': np.int,
+               'int4': np.int,
+               'int8': np.int,
+               'char': np.str,
+               'character': np.str,
+               'nchar': np.str,
+               'bpchar': np.str,
+               'varchar': np.str,
+               'character varying': np.str,
+               'nvarchar': np.str,
+               'text': np.str}
```

Comment:
We might want to pick the numpy type that is appropriately sized for the redshift type. For example, for 'float8' we should prob use `np.float64`, etc. Ditto for the various int types and the rest of the float types.

Reply:
I had considered that but I've found in a private repo of ours that has similar functionality and they convert all numerical to np.float but I thought it would be fine to leave the ints.

Reply:
I think here we want to preserve the type sizes properly since we don't know what users are doing with the data. We can chat about the other stuff offline. I think I understand the confusion.